test/test_helper.rb

```diff
@@ -0,0 +1,27 @@
+ENV["Y2DIR"] = File.expand_path("../../src", __FILE__)
+
+require "yast"
+require "yast/rspec"
+
+# force utf-8 encoding for external
+Encoding.default_external = Encoding::UTF_8
```

Comment:
I think you could alternatively use `ENV["LC_ALL"] = "en_US.UTF-8"` which has an advantage that the translated messages do not break the tests when running in non-English locale.